scripts/automated_ingestion/ingest_bundles.py

```diff
@@ -0,0 +1,263 @@
+    parser.add_argument("-d", "--debug", help="enable debug mode", action="store_true", dest="debug")
+    parser.add_argument("-l", "--list", help="only list available tasks", action="store_true", dest="list_only")
+    parser.add_argument("--extensions", help="comma-separated list of extensions to process (default: .task)",
+                        nargs="?", const=".task", default=False)
+
+    return parser.parse_args()
+
+
+@log_function_entry_exit()
+def setup_logging(config: configparser.ConfigParser, args: argparse.Namespace) -> logging.Logger:
+    """
+    Configure logging based on configuration file and command line arguments.
+    Command line arguments take precedence over config file settings.
+
+    Args:
+        config: Configuration parser
+        args: Parsed command line arguments
+
+    Returns:
+        Logger instance
+    """
+    # get settings from config file
+    log_file = config["logging"].get("log_file")
+    config_console_level = LOG_LEVELS.get(config["logging"].get("console_level", "INFO").upper(), logging.INFO)
+    config_file_level = LOG_LEVELS.get(config["logging"].get("file_level", "DEBUG").upper(), logging.DEBUG)
+
+    # override with command line arguments if provided
+    log_file = args.log_file if args.log_file else log_file
+    console_level = getattr(logging, args.console_level) if args.console_level else config_console_level
+    file_level = getattr(logging, args.file_level) if args.file_level else config_file_level
+
+    # debug mode overrides console level
+    if args.debug:
+        console_level = logging.DEBUG
+
+    # set up logging scopes
+    if args.log_scopes:
+        set_logging_scopes(args.log_scopes)
+        log_message(LoggingScope.DEBUG, "DEBUG", "Enabled logging scopes: '%s'", args.log_scopes)
+
+    # create logger
+    logger = logging.getLogger()
+    logger.setLevel(logging.DEBUG)  # set root logger to lowest level
+
+    # create formatters
+    console_formatter = logging.Formatter("%(levelname)-8s: %(message)s")
+    file_formatter = logging.Formatter("%(asctime)s - %(levelname)-8s: %(message)s")
+
+    # console handler (only if not quiet)
+    if not args.quiet:
+        console_handler = logging.StreamHandler(sys.stdout)
+        console_handler.setLevel(console_level)
+        console_handler.setFormatter(console_formatter)
+        logger.addHandler(console_handler)
+
+    # file handler (if log file is specified)
+    if log_file:
+        # ensure log directory exists
+        log_path = Path(log_file)
+        log_path.parent.mkdir(parents=True, exist_ok=True)
+
+        file_handler = logging.FileHandler(log_file)
+        file_handler.setLevel(file_level)
+        file_handler.setFormatter(file_formatter)
+        logger.addHandler(file_handler)
+
+    return logger
+
+
+@pidfile("shared_lock.pid")  # noqa: F401
+@log_function_entry_exit()
+def main():
+    """Main function."""
+    args = parse_args()
+    config = parse_config(args.config)
```

Comment:
This may be my lack of knowledge, but: is there an example config file? Or documentation on what the config files should look like (i.e. which keywords exist, what they mean, what valid values are, etc).